app/src/shared/components/EditableText/editableText.pcss

```diff
@@ -30,13 +30,25 @@
   background-color: #CCE9FD;
 }
 
+.idle {
+  cursor: text;
+}
```

Comment:
I note that this cursor looks weird when dragging the module around by the header. Maybe the editable text's width in the header should be `min-content` or perhaps the cursor should get changed if a drag is in progress.

Reply:
https://streamr.atlassian.net/browse/PLATFORM-533